styles/components/blog.scss

```diff
@@ -0,0 +1,52 @@
+.container {
```

Comment:
Watch out, you are redeclaring rules for `.container` multiple places!

This is all of the styles needed for container. The use of `.container` on the home page is doing to much and you should refer back to the https://online.theironyard.com/paths/579/units/3686/lessons/15005 lesson about the purpose of sections, grids, and containers.